pyqtgraph/parametertree/Parameter.py

```diff
@@ -10,6 +10,24 @@
 PARAM_TYPES = {}
 PARAM_NAMES = {}
 
+_PARAM_ITEM_TYPES = {}
+
+def registerParameterItemType(name, itemCls, parameterCls=None, override=False):
```

Comment:
This is nice :+1:

It'd be good to add this to `parametertree/__init__.py` and include it in the docs similar to `registerParameterType`.